app/views/ProfileView/index.tsx

```diff
@@ -266,6 +280,12 @@ const ProfileView = ({ navigation }: IProfileViewProps): React.ReactElement => {
 		navigation.setOptions(options);
 	}, []);
 
+	useFocusEffect(
+		useCallback(() => {
+			reset();
+		}, [])
+	);
```

Comment:
I think this might be wrong.
The goal seems to be to reset the form when ProfileView is focused, but I wonder what's the behavior when the user navs to `edit avatar` or `preferences` and then navs back to ProfileView.